README.md

```diff
@@ -5,6 +5,15 @@
 
 [ReactPHP](https://reactphp.org/)'s core reactor event loop that libraries can use for evented I/O.
 
+> **Development version:** This branch contains the code for the upcoming
+> v3 release. For the code of the current stable version 1.5 release, check
```

Comment:
How about using v1 instead if 1.5 here? It's possible that, while working on v3, we will release a v1.6 for reactphp/event-loop. This would then lead to us also updating the documentation here, which is not necessary if we avoid specifying a certain version.

```suggestion
> v3 release. For the code of the current stable v1 release, check
```